test/distributed/org/apache/cassandra/distributed/upgrade/UpgradeTestBase.java

```diff
@@ -111,14 +110,14 @@ public UpgradeableCluster.Builder builder()
     public static class TestVersions
     {
         final Version initial;
-        final List<Version> upgrade;
-        final List<Semver> upgradeVersions;
+        final Version targetVersion;
+        final Semver targetSemver;
```

Comment:
these lists always had length 1.
Is there a value in retaining the lists?

Reply:
they need to be lists.

Reply:
why?